### ISSUE TITLE
Discord RPC not working but there are no errors

### ISSUE THREAD

**Issue by ZroNoh** [Ombrelin/plex-rich-presence#31]
I don't have any errors to show as it gives me nothing, and everything seems to be setup correctly on my end. I've tried running both the jar file and the windows installer but neither seem to work.

no errors going on
![javaw_4k0TlMr9HU](https://user-images.githubusercontent.com/32776743/109374976-36c91d00-7887-11eb-90fd-fa87b7059959.png)

the program is in my authorized devices
![brave_c308K8o7vL](https://user-images.githubusercontent.com/32776743/109374972-2dd84b80-7887-11eb-8c1b-241fd7ced849.png)

im actually playing something
![brave_oMpV5LtMGw](https://user-images.githubusercontent.com/32776743/109374981-3fb9ee80-7887-11eb-9882-44194f9f7e40.png)

game activity is on
![Discord_dVKjKd5z1T](https://user-images.githubusercontent.com/32776743/109375010-5eb88080-7887-11eb-8c50-7a1ceab84753.png)

but its not detecting it
![Discord_L3mqfIFmLK](https://user-images.githubusercontent.com/32776743/109375012-61b37100-7887-11eb-86d6-6e120eb60bc7.png)

**Comment by mclarence** [comment]
Please follow steps here : https://github.com/Ombrelin/plex-rich-presence/issues/9#issuecomment-773108634

Post log here.

**Comment by ZroNoh** [comment]
[log.txt](https://github.com/Ombrelin/plex-rich-presence/files/6055179/log.txt)

**Comment by Ombrelin** [comment]
Mmmh, seems like you have no server on your account from which you're owner.

**Comment by Ombrelin** [comment]
Discord Rich Presence only works for server on which you're owner

**Comment by ZroNoh** [comment]
i have a server though, I use it every day?

**Comment by mclarence** [comment]
https://github.com/Ombrelin/plex-rich-presence/issues/30

Read discussion there and let me know if it fixed issue.

**Comment by Ombrelin** [comment]
Yeah but you're not the Owner of the server (user with the crown icon when connecting).
Plex Rich Presence uses the current active sessions on a server which is only accessible to the owner

**Comment by mclarence** [comment]
> Yeah but you're not the Owner of the server (user with the crown icon when connecting).
> Plex Rich Presence uses the current active sessions on a server which is only accessible to the owner

It does appear he is the owner of the Plex server according to the screenshots  provided.

**Comment by Ombrelin** [comment]
Ha yes indeed thank you for your insight. I'm a bit confused be the log then which state he owns none of the servers

**Comment by ZroNoh** [comment]
I have my own server but my account was added to a Plex Home. The owner has their own server as well. Could this be part of it?

**Comment by ZroNoh** [comment]
I also have two friends sharing their libraries with me

**Comment by mclarence** [comment]
@ZroNoh  Please read this issue here #30

This user had a similar issue. It was fixed by either changing the port numbers so that external and internal port numbers are the same or by updating the plex server to the latest version.

Let me know how it goes.

**Comment by mclarence** [comment]
Please download the new version released and manually specify your plex server in the settings.

Let me know how it goes.
